source/includes/_ingress.md

```diff
@@ -101,16 +111,21 @@ curl https://api.onramp.ltd/rpc/create_ingress_invoice
-timeout_in_sec    | Integer         | When to expire the link if unused.                                                                                                   | Yes
-offer_skin        | Ingress Skin    | Specify how the offer should be displayed to the user.                                                                               | Yes
-billing_details   | Billing Details | User billing details. Please, notice how **not** all parameters inside this json object are required except `merchant_customer_id`.  | Yes
-cancel_callback   | Url             | Merchant callback endpoint to be called when an ingress payment couldn't be completed                                                | No
+Field                    |   Type          | Description                                                                                                                          | Required
+-----------------        | --------------- | ------------------------------------------------------------------------------------------------------------------------------------ | --------
+fiat_amount              | Integer         | Amount to be paid denominated in cents .                                                                                             | Yes
+fiat_currency            | String          | Currency identifier following the [ISO 4217 standard](https://en.wikipedia.org/wiki/ISO_4217). Valid values are `"EUR"` or `"USD"`.  | Yes
+payment_ack_url          | Url             | Merchant callback endpoint to confirm ingress transaction. It should be a complete, well formed, url.                                | Yes
+user_redirect_url        | Url             | Where to redirect the user after a successful payment. It should be a complete, well formed, url.                                    | Yes
+timeout_in_sec           | Integer         | When to expire the link if unused.                                                                                                   | Yes
+offer_skin               | Ingress Skin    | Specify how the offer should be displayed to the user.                                                                               | Yes
+billing_details          | Billing Details | User billing details. Please, notice how **not** all parameters inside this json object are required except `merchant_customer_id`.  | Yes
+cancel_callback          | Url             | Merchant callback endpoint to be called when an ingress payment couldn't be completed.                                               | No
+card_number              | String          | Credit card number to be used to attempt the payment. If not sent, no credit card details will be prefilled.                         | No       
+card_expiry_date         | String          | Credit card expiry date to be used to attempt the payment.                                                                           | No
```

Comment:
Would be helpful to specify exact format as it may vary between regions (like `YYYY-MM` vs `MM/YYYY` etc)

Reply:
Yes, I think overall we should a format column on the tables. I'll add this in another PR. For the moment, the example values should be ok.